src/master/master-dashboard/master_dashboard.jsx

```
@@ -563,252 +695,124 @@ function MasterDashboard() {
-                  {selectedOrganizationIds.length > 0 && (
-                    <button
-                      onClick={bulkDelete}
-                      className={btnRoseClass}
-                      disabled={isLoading}
-                    >
-                      <i className="fas fa-trash-alt"></i>
-                      Delete ({selectedOrganizationIds.length})
-                    </button>
-                  )}
+          {/* Reusable Filter Component */}
+          // Reusable Filter Component
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)